ESMF/Apps/esmApp.F90

```diff
@@ -0,0 +1,80 @@
+!==============================================================================
```

Comment:
Why is this file here?   It does not seem to use any GEOS code.

Reply:
Will remove